src/components/Events.tsx

```
@@ -0,0 +1,41 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import type { Event } from "../model/Event";
+import style from "./Events.module.scss";
+import { Loader } from "../components/Loader";
+
+export const Events: React.FC = () => {
+  const [events, setEvents] = useState<Event[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchEvents = async () => {
+      try {
+        const response = await fetch(
+          "https://sportujspolu-api.onrender.com/api/v1/events"
+        );
+        setEvents(await response.json());
+        setIsLoading(false);
+      } catch (error) {}
```

Is the empty catch necessary? Just remove it

---

updated